### PR TITLE
Feature/FOR-911: All Always label all form fields.

### PR DESCRIPTION
**Pull request by edwinanciani**
Changes to be committed:
	modified:   src/components/address.js
	modified:   src/components/currency.js
	modified:   src/components/datagrid.js
	modified:   src/components/datetime.js
	modified:   src/components/day.js
	modified:   src/components/editgrid.js
	modified:   src/components/file.js
	modified:   src/components/number.js
	modified:   src/components/password.js
	modified:   src/components/phonenumber.js
	modified:   src/components/radio.js
	modified:   src/components/resource.js
	modified:   src/components/select.js
	modified:   src/components/selectboxes.js
	modified:   src/components/survey.js
	modified:   src/components/textarea.js
	modified:   src/components/textfield.js
	modified:   src/components/time.js
	modified:   src/factories/BuilderUtils.js

### CODE REVIEW

File: src/factories/BuilderUtils.js [hunk 1]
```diff
@@ -73,6 +73,25 @@ module.exports = ['FormioUtils', function(FormioUtils) {
...
+  /**
+   * Iterate the given label to make it unique.
+   *
+   * @param {String} label
+   *   Modify the component label to be unique.
+   *
+   * @returns {String}
+   *   The new component label.
+   */
+  var iterateLabel = function(label) {
```
Comment (travist) [inline src/factories/BuilderUtils.js]: What if you have two "First Name" fields, but one of them is in a "Customer" container. This should be acceptable. Will this instead force them to make it say "First Name 2"?

Comment (randallknutson) [reply src/factories/BuilderUtils.js]: Do labels have to be unique? I didn't think that was a constraint.

Comment (travist) [reply src/factories/BuilderUtils.js]: My complaint exactly. They should not have to be unique.

Comment (edwinanciani) [reply src/factories/BuilderUtils.js]: Actually yes, you are right!

Comment (edwinanciani) [reply src/factories/BuilderUtils.js]: The only change is in ngFormio

**Pull request closed without merging**